lib/stats.rb

```diff
@@ -185,6 +186,13 @@ def eight_am(day)
     Time.new(day.year, day.month, day.day, 8)
   end
 
+  def quota
```

Comment:
Would you mind adding a simple test to `test/stats_test.rb` that describes an expected result here?